pkg/buildpipeline/pipeline_test.go

```diff
@@ -1029,6 +1031,7 @@ func TestParseJenkinsfileYaml(t *testing.T) {
 			}
 
 			if tt.expectedErrorMsg == "" {
+				pipeline.TypeMeta = metav1.TypeMeta{}
```

Comment:
I started setting `TypeMeta` on the generated CRDs so we don't have to set it later when applying them to the cluster (maybe I shouldn't do that?) but realized it would require updating all of the tests to accommodate and didn't want to create more conflicts in the short term. Happy to adjust to actually setting the values in the tests if we want that, or to not set TypeMeta until later if we want that instead.

Reply:
I honestly don't know what the right answer here is, so this should be fine for right now.